### PR TITLE
Add i686-pc-windows-msvc in release workflow

### LINKED ISSUE #419: Adding more pre-built platforms
Hello, 

Would it be possible / reasonable to add  `i686-pc-windows-msvc` and `x86_64-unknown-linux-musl` to the pre-built targets ?
This would ease our process of update `cargo-nextest` in our CI.

Thanks.

### PR DESCRIPTION
**Pull request by Guiguiprim**
Closes #419

- Site: Add latest pre-built link for `x86_64-unknown-linux-musl`
- Release: Add pre-build for `i686-pc-windows-msvc`
- Site: Add info and links for pre-built for `i686-pc-windows-msvc`

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -213,6 +219,8 @@ jobs:
             --alias linux-arm=aarch64-unknown-linux-gnu:tar.gz \
             --alias windows=x86_64-pc-windows-msvc:zip \
             --alias windows-tar=x86_64-pc-windows-msvc:tar.gz \
+            --alias windows-32=i686-pc-windows-msvc:zip \
+            --alias windows-32-tar=i686-pc-windows-msvc:tar.gz \
```
Comment (sunshowers) [inline .github/workflows/release.yml:223]: Thanks! Can we rename these to `windows-x86` and `windows-x86-tar`?

**Review by sunshowers — CHANGES_REQUESTED** [review]
Looks good overall :) just one change requested.

File: .github/workflows/release.yml (hunk 1 above)
Comment (Guiguiprim) [reply .github/workflows/release.yml]: Done

**Review by sunshowers — APPROVED** [review]
Looks good -- I'll land it as I'm about to release the next version.

**Comment by sunshowers** [comment]
@Guiguiprim this is now out in cargo-nextest 0.9.32. Thanks again!

**Pull request merged**